go/oasis-test-runner/oasis/client.go

```diff
@@ -23,6 +23,7 @@ func (client *Client) startNode() error {
 	args := newArgBuilder().
 		debugDontBlameOasis().
 		debugAllowTestKeys().
+		workerCertificateRotation(true).
```

Comment:
The client doesn't actually register, so this is not needed?

Reply:
This is currently consistent with previous code: https://github.com/oasisprotocol/oasis-core/pull/3062#discussion_r447062679 but yeah can remove it